bindings/python/scripts/robot_wrapper.py

```diff
@@ -18,7 +18,7 @@ def BuildFromURDF(filename, package_dirs=None, root_joint=None, verbose=False):
         robot.initFromURDF(filename, package_dirs, root_joint, verbose)
         return robot
 
-    def initFromURDF(self,filename, package_dirs=None, root_joint=None, verbose=False):
+    def initFromURDF(self,filename, package_dirs=None, root_joint=None, verbose=False, meshLoader=None):
```

Comment:
You must handle the case where Pinocchio is no built with `hpp-fcl`support.

Reply:
What do you expect me to do here. Raise an exception is meshLoader is provided and not built with hpp-fcl ?
If so, how do you get the information about hpp-fcl support in Python ?

Reply:
One way is to check if hpp-fcl is provided, I was planning to do that by adding a global variable called maybe `__PINOCCHIO_WITH_HPP_FCL__`.
In the RobotWrapper, we have a similar strategy but checking if https://github.com/stack-of-tasks/pinocchio/blob/36aeeebadfe1c55675f2265bdb8fe534d50e95cb/bindings/python/scripts/robot_wrapper.py#L27 if a function is included inside the namespace `pinocchio`.

Reply:
As far as I understand, "buildGeomFromUrdf" is always part of pinocchio in Python, regardless the use of hpp-fcl. (even before this PR). So I guess this test is always false.